bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/connection/Clip2Bridge.java

```diff
@@ -397,7 +398,8 @@ public void onFailure(@Nullable Session session, @Nullable Throwable failure) {
         public void onGoAway(@Nullable Session session, @Nullable GoAwayFrame frame) {
             Objects.requireNonNull(session);
             if (session.equals(http2Session)) {
-                Thread recreateThread = new Thread(() -> recreateSession());
+                Thread recreateThread = new Thread(() -> recreateSession(),
+                        "OH-binding-" + HueBindingConstants.BINDING_ID + "-Clip2Bridge");
```

Comment:
Probably we could use the `getThing.getUID()` here? It would help in the (admittedly seldom) case that there are to different bridges..

Reply:
```
Thread recreateThread = new Thread(() -> recreateSession(),
    "OH-binding-" + Clip2Bridge.this.bridgeHandler.getThing().getUID());
```

Reply:
Updated, thanks